rxjava-support/src/test/java/tech/picnic/rx/PicnicRxPluginsTest.java

```diff
@@ -50,14 +50,15 @@ public void testPropagate() {
     verifyVerificationCounter(ctx2, 3);
 
     // With context propagation
-    init();
+    setUp();
```

Comment:
It's a bit wonky that we call this method in the middle of a test; should be refactored in the context of another PR.

Reply:
👍 Is the rename following Junit standards?

Reply:
Yep, it's what one of the PMD rules enabled by default in Codility suggested. (Now it still complains about a bunch of other violations we don't care about; will review that before merging this PR.)